infrastructure/kube/keep-prd/btc-difficulty/config/btc-difficulty-maintainer-keyfile

```diff
@@ -0,0 +1,2 @@
+// TODO: Fill with proper values for account `0xCb6Ed7E78d27FDff28127F9CbD61d861F09a2324`
```

Comment:
We need to provide the proper keyfile for the maintainer account (`0xCb6Ed7E78d27FDff28127F9CbD61d861F09a2324`).